src/site/content/en/fast/codelab-replace-gifs-with-video/index.md

```diff
@@ -155,11 +155,32 @@ Update the `<video>` with `<source>` elements that link to your cat-herd videos:
 
 {% Aside %}
 Browsers don't speculate about which `<source>` is optimal, so the order
-of `<source>`s matters. For example, if you specify an MP4 video first and the
+of `<source>` elements matters. For example, if you specify an MP4 video first and the
 browser supports WebM, browsers will skip the WebM `<source>` and use the MPEG-4
 instead. If you prefer a WebM `<source>` be used first, specify it first!
 {% endAside %}
 
+## Add captions and descriptions
+One last detail to take care of: accessibility.
+With GIFs, you can provide alternative text
+to describe the image to users with visual impairments.
+For videos, you need two elements:
+- A `<track kind="descriptions">` element that links to a [WebVTT](https://developer.mozilla.org/en-US/docs/Web/API/WebVTT_API)
```

Comment:
Since this is a codelab can we show folks what to put in these files?

I also wanted to double check that these don't automatically force the captions to show on screen.

Reply:
First thing: I suppose so. WebVTT is sufficiently complicated that I was hesitating to go down that rabbit hole because it could make the codelab a lot longer. Were you thinking of just showing the VTT and saying "go look at the MDN page to understand what all this is"?

Second thing: captions don't automatically show.

Reply:
> Were you thinking of just showing the VTT and saying "go look at the MDN page to understand what all this is"?

yup